.github/workflows/build-os.yml

```diff
@@ -144,7 +144,7 @@ jobs:
           run: |
             export DEBIAN_FRONTEND=noninteractive
             sudo apt-get update -y -o Dpkg::Progress-Fancy=0
-            sudo apt-get dist-upgrade -y -o Dpkg::Progress-Fancy=0
+            sudo apt-get full-upgrade -y -o Dpkg::Progress-Fancy=0
```

Comment:
I wonder if it might be more logically organized to run the apt-get upgrade command in https://github.com/PlanktoScope/PlanktoScope/blob/master/software/distro/setup/setup.sh rather than this "Record OS installation versioning information" step of the CI workflow?